analyzebooks.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python3
+'''
+    DESC: Creates database of Charles Dickens books and compares unknown books
+    to database to determine whether or not Charles Dickens is the author
+    AUTH: Connor Novak
+    MAIL: connor.novak@students.olin.edu
+    '''
+
+import string
+import operator
+import numpy as np
+import pdb
+
+def get_word_list(file_name):
+    '''
+        DESC: Reads specified project Gutenberg book, strips header comments,
+        punctuation, and whitespace, then returns lowercase list of words in book.
+        ARGS:
+        file_name - str - name of file where Gutenberk book text is stored
+        RTRN: list of lowercase words in book
+        '''
+    book = open(file_name, 'r')
+    lines = book.readlines()
+
+    curr_line = 0
+    # For every line that is before the start of the book, cut it out of lines
+    while lines[curr_line].find('START OF THIS PROJECT GUTENBERG EBOOK') == -1:
+        curr_line += 1
+    lines = lines[curr_line+1:]
+
+    curr_line = len(lines)
+    # For every line that is after the end of the book, cut it out of lines
+    while lines[curr_line-1].find('END OF THIS PROJECT GUTENBERG EBOOK') == -1:
+        curr_line -= 1
+    lines = lines[:curr_line-1]
+
+    # creates list of all words in all lines
+    words = []
+    for line in lines:
+        line = line.replace('\u2019','')
+        line = line.translate(str.maketrans('\u201c' + '\u201d' +string.punctuation,"                                  "))
```

Comment:
This is not a common operation so an in-line comment describing what this operation does would be helpful.